setup.py

```diff
@@ -6,16 +6,23 @@
     packages=['mycroft_voice_satellite', 'mycroft_voice_satellite.speech',
               'mycroft_voice_satellite.res',
               'mycroft_voice_satellite.configuration'],
-    install_requires=["jarbas_hive_mind>=0.10.3",
-                      "speech2text",
+    install_requires=["jarbas_hive_mind",
+                      "SpeechRecognition @ git+https://github.com/Uberi/speech_recognition.git",
```

Comment:
why is this needed?

Reply:
Removed the git require as it is not needed.